src/components/training/UpcomingTrainingSection.js

```diff
@@ -52,7 +48,10 @@ const CorporateCrossSell = styled.div`
 `
 
 export const UpcomingTrainings = ({ curriculum, type, trainings }) => {
-  const filteredTrainings = selectUpcomingTrainings({ type, trainings })
+  const filteredTrainings = getUpcomingTrainingsByType({
+    types: [...type],
```

Comment:
@ric9176 do we have to spread the type? isn't it a string? We should probably add prop types to all the components to avoid this kind of confusion. If the prop `type` is an array of types, then I think `types` make more sense than type

Reply:
In future it will just be a string, this is just to solve our issue that we currently have a React workshop using the ONE_DAY_WORKSHOP type that will be deprecated and that needs to be displayed along with the new ones with the correct type of REACT_WORKSHOP. We discussed in this in the review of the sprint.

Reply:
So I’m just one vas it’s an array with more than one value, in all others we are passing that function and array with one item, so this means it can for now handle both but it’s an edge case that won’t exist in future so can be refactored back